web/docs/setup/tune.md

```diff
@@ -333,3 +306,41 @@ vast rebuild start '#type == "suricata.flow" && #import_time < 2 weeks ago'
 ```
 
 To stop an ongoing rebuild, use the `vast rebuild stop` command.
+
+## Shutdown
+
+The `stop` command gracefully brings down a VAST server that has been started
+with the `start` command.
+
+It is also possible to send a signal `SIGINT(2)` to the `vast` process instead
+of using `vast stop`, but in only works on the same machine that runs the
+server process. We recommend using `vast stop`, as it also works over the wire.
+
+The `stop` command blocks until the server process has terminated, and returns
+a zero exit code upon success, making it suitable for use in launch system
+scripts.
+
+:::note
+The server waits for ongoing import processes to terminate before shutting down
+itself. In case an import process is hanging, you can always terminate the
+hanging process manually to shutdown the server.
+:::
```

Comment:
This is more of a "known bug" that we simply haven't been able to fix, but hopefully will go away implicitly with the pipeline-first efforts. I think we should rephrase it or remove it.